src/postgraphql/postgraphql.ts

```diff
@@ -26,6 +27,7 @@ type PostGraphQLOptions = {
   exportJsonSchemaPath?: string,
   exportGqlSchemaPath?: string,
   bodySizeLimit?: string,
+  pgTransactionSetup?: (opts: TransactionSetupOpts) => Promise<string | undefined>,
```

Comment:
This needs a much scarier name like `dangerouslySetInnerHTML` has in React; maybe something like: `pgDangerouslyOverrideDefaultTransactionSetup`

We should also throw an error if you specify any of the options that `setupPgClientTransaction` uses if you are using `pgDangerouslyOverrideDefaultTransactionSetup`, e.g. `jwtRole`, `jwtSecret`, `jwtAudiences`, `pgDefaultRole`, `pgSettings` - because you're opting to take this entirely into your own hands you will be responsible for re-implementing these functionalities and supporting the default options will be misleading.